### PR TITLE
Added Jenkins file

### PR DESCRIPTION
**Pull request by gustavonalle**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                        junit testDataPublishers: [[$class: 'ClaimTestDataPublisher']], testResults: '**/target/*-reports*/*.xml'
+                        sh "${mvnHome}/bin/mvn clean -s $MAVEN_SETTINGS"
+                    }
+                }
+            }
+        }
+
+        stage('X-Site tests') {
+            when {
+                branch 'master'
```
Comment (slaskawi) [inline Jenkinsfile]: Could you change this to 8.2.x?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                        junit testDataPublishers: [[$class: 'ClaimTestDataPublisher']], testResults: '**/target/*-reports/*.xml'
+                        sh "${mvnHome}/bin/mvn clean -s $MAVEN_SETTINGS"
+                    }
+                }
+            }
+        }
+        
+        stage('Deploy SNAPSHOT') {
+            when {
+                branch 'master'
```
Comment (slaskawi) [inline Jenkinsfile]: And here?

**Comment by gustavonalle** [comment]
@slaskawi done

**Comment by slaskawi** [comment]
Thanks @gustavonalle !

**Pull request merged**